src/media-group-controller.ts

```diff
@@ -306,7 +306,7 @@ export class MediaGroupController extends EventTarget {
         // https://www.prosoundtraining.com/site/wp-content/uploads/2014/02/Lip-Sync-Errors.pdf
         // Safari is very difficult to get synced so close.
         // Working with playbackRate is impossible on Safari, try with seeking.
-        const target = child.dataset.groupSeekPrecision ?? 0.05;
```

Comment:
thanks for the contribution! 

I made this to be configurable, see the dataset prop. 
it corresponds with the `data-group-seek-precision` attribute on the follower media element.

Reply:
Yes, I noticed that, but where you able to reproduce in Safari? don't you think it be better if the default has the library working smoothly in safari out of the box? and then it can still be configured as a last resort? 
Or there's some other reason to keep it at `0.05` even if it's not buggy in safari?

Reply:
you're right. I haven't tested it out this time around.

I remembered from when I created this lib that value was need to get a higher precision with decent performance.

Reply:
thinking a bit, a 0.5s for video to audio is quite a lot. think of lip sync etc. 

what is your use case if I could ask?

Reply:
I am open to a lower value, I came to `0.5s` by trying out values above and below and seeking in the example page on safari, until I found one that didn't cause glitches when moving the playhead.

Re  lip sync, good point, is `groupSeekPrecision` used also during "regular" playing or just when the playhead is moved during a seek event?

If it's only used during seek event triggered by user scrubbing on the progress bar then that shouldn't be an issue? (altho let me know if I am overlooking something here)

Reply:
The use case is broadly synchronizing an audio track with a video track and being able to toggle the audio between the two.

Reply:
yes, you see the timing was chosen based on this doc 
https://www.prosoundtraining.com/site/wp-content/uploads/2014/02/Lip-Sync-Errors.pdf

the "glitches" are timed out repeating seeks until the target of <=0.05s is met.